src/ExchangeRate.java

```diff
@@ -0,0 +1,219 @@
+
+import components.simplereader.SimpleReader;
+import components.simplereader.SimpleReader1L;
+import components.simplewriter.SimpleWriter;
+import components.simplewriter.SimpleWriter1L;
+
+/**
+ * {@code ExchangeRate} represented as a {@code Double} with implementations of
+ * primary methods.
+ *
+ * @convention <pre>
+ * [all characters of $this.rep are '0' through '9']
+ * </pre>
+ * @correspondence <pre>
+ * this = [the decimal number whose ordinary depiction is $this.rep]
+ * </pre>
+ *
+ * @author Ryan Ouimet
+ *
+ */
+
+public class ExchangeRate {
+
+    /*
+     * Private members --------------------------------------------------------
+     */
+
+    /**
+     * Representation of {@code this}.
+     */
+    private Double rep;
```

Comment:
Are you planning to store any additional information, like the names of the currency?